### PR TITLE
Add Footer and fontface

### PR DESCRIPTION
**Pull request by danieledamiani**

### CODE REVIEW

File: packages/website_v2/src/components/Footer/Footer.scss [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    width: 100%;
+    padding-left: 0;
+    margin: 0;
+
+    > li {
+      display: inline-block;
+      margin-right: 40px;
+      
+      > a {
+        color:#EEECFF;
```
Comment (iamvukasin) [inline packages/website_v2/src/components/Footer/Footer.scss:36]: Space between : and value

File: packages/website_v2/src/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          </a>
+        </li>
+        <li>
+          <a href="https://tempus.finance/disclaimer" target="_self">
+            disclaimer
+          </a>
+        </li>
+        <li>
+          <a href="https://tempus.finance/privacy-policy" target="_self">
+            {' '}
```
Comment (iamvukasin) [inline packages/website_v2/src/components/Footer/Footer.tsx:25]: Why do we have this space?

File: packages/website_v2/src/components/App/App.scss [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+@font-face {
+  font-family: articulat-cf, sans-serif;
+  font-weight: 700;
+  font-style: normal;
+  src: url('https://use.typekit.net/fgw2twm.css') format('truetype');
+}
+
+@font-face {
+  font-family:  articulat-heavy-cf, sans-serif;
```
Comment (iamvukasin) [inline packages/website_v2/src/components/App/App.scss:16]: Remove double space

**Review by iamvukasin — APPROVED** [review]

File: packages/website_v2/src/components/Footer/Footer.tsx (hunk 2 above)
Comment (danieledamiani) [reply packages/website_v2/src/components/Footer/Footer.tsx:25]: Autoformatting issue

File: packages/website_v2/src/components/App/App.scss (hunk 3 above)
Comment (danieledamiani) [reply packages/website_v2/src/components/App/App.scss]: fixed

**Pull request merged**